gsbparse/account_section.py

```diff
@@ -405,7 +405,7 @@ def __init__(self, XML_tags_attributes_values):
             XML_tags_attributes_values (list(dict)): Values of "Transaction"
```

Comment:
Could you please also fix the typo on this line?

Reply:
Fixed, I split the commit in two, one for typos, the other for the payment section :)